CHANGELOG.md

```diff
@@ -3,7 +3,13 @@
 
 ## New Features:
 
-- Add keyword argument `elem_classes` to Components to control class names of components, in the same manner as existing `elem_id`. 
+### Official Theme release
+
+Gradio now supports a new theme system, which allows you to customize the look and feel of your app. You can now use the `theme=` kwarg to pass in a prebuilt theme, or customize your own! See guides/03_building-with-blocks/06_theming-guide for more details. By [@aliabid94](https://github.com/aliabid94) in [PR 3470](https://github.com/gradio-app/gradio/pull/3470) 
```

Comment:
I don't think the Theming Guides should live in in the Blocks section. It applies equally to Interfaces and people might not realize that if we put it in Blocks. How about moving it to Other Tutorials?